docs/identity/saas-apps/officespace-tutorial.md

```diff
@@ -105,11 +78,11 @@ Follow these steps to enable Microsoft Entra SSO.
 
 	![Edit SAML Signing Certificate](common/edit-certificate.png)
 
-1. In the **SAML Signing Certificate** section, copy the **Thumbprint Value** and save it on your computer.
+1. In the **SAML Signing Certificate Section**, choose to choose to download the PEM certificate.
```

Comment:
Is the extra "choose to" an intended repetition?

Reply:
Apologies, but this is a typo. If you could remove the extra choose to, I would appreciate it. Should be:

1. In the **SAML Signing Certificate Section**, choose to download the PEM certificate.